pkg/apis/kops/componentconfig.go

```diff
@@ -207,6 +207,8 @@ type KubeletConfigSpec struct {
 	CgroupDriver string `json:"cgroupDriver,omitempty" flag:"cgroup-driver"`
 	// HousekeepingInterval allows to specify interval between container housekeepings.
 	HousekeepingInterval *metav1.Duration `json:"housekeepingInterval,omitempty" flag:"housekeeping-interval"`
+	// EventQPS if > 0, limit event creations per second to this value.  If 0, unlimited.
+	EventQPS *int32 `json:"eventQPS,omitempty" flag:"event-qps" flag-empty:"0"`
```

Comment:
I think `event-burst` should also be added together with `event-qps`.
https://github.com/kubernetes/kubernetes/blob/87984d84d18c7816835e5efedebb17ed13aa2509/cmd/kubelet/app/options/options.go#L468

Reply:
Done

Reply:
Could you run `make apimachinery` once more, please?